api/HTMLOptionsCollection.json

```diff
@@ -61,10 +61,10 @@
               "version_added": "≤18"
             },
             "firefox": {
-              "version_added": true
+              "version_added": "4"
```

Comment:
Looks like `add` and `remove` were added slightly after the main interface, but that's reasonable, it started out as a readonly representation I guess.